custom_components/kidde/binary_sensor.py

```diff
@@ -34,6 +34,28 @@
         name="CO Alarm",
         device_class=BinarySensorDeviceClass.CO,
     ),
+    BinarySensorEntityDescription(
+        "hardwire_smoke",
+        icon="mdi:smoke-detector-variant-alert",
+        name="Hardwire Smoke Alarm",
+        device_class=BinarySensorDeviceClass.SMOKE,
+    ),
+    BinarySensorEntityDescription(
+        "too_much_smoke",
+        icon="mdi:smoke-detector-variant-alert",
+        name="Too Much Smoke",
+        device_class=BinarySensorDeviceClass.SMOKE,
+    ),
```

Comment:
Should we put `too_much_smoke` into the Diagnostic category?

```suggestion
    BinarySensorEntityDescription(
        "too_much_smoke",
        icon="mdi:smoke-detector-variant-alert",
        name="Too Much Smoke",
        device_class=BinarySensorDeviceClass.SMOKE,
        entity_category=EntityCategory.DIAGNOSTIC,
    ),
```

Reply:
I could argue both keeping it `Normal` or going with `Diagnostic`. 
- `Normal`: used to alert 
- `Diagnostic`: used to further diaganose `smoke_alarm`